main.gms

```diff
@@ -509,6 +509,15 @@ parameter
 *' * (-1): default setting equivalent to no carbon tax
 *' * (any number >= 0): tax level in 2020, with 5% exponential increase over time
 *'
+parameter
+  cm_co2_tax_startyear    "level of co2 tax in start year in $ per t CO2eq"
+;
+  cm_co2_tax_startyear = -1;     !! def = -1  !! regexp = -1|is.nonnegative
+*' * Parameter for realizations 'diffLin2Lin' and 'diffExp2Lin' of 45_carbonprice
+*' * (-1): default setting equivalent to no carbon tax
+*' * (any number >= 0): co2 tax in start year [if cm_iterative_target_adj eq 0]; 
+*' *                    exogenous initialization of co2 tax in start year [if cm_iterative_target_adj ne 0]
+*'
```

Comment:
Introducing new switch cm_co2_tax_startyear to avoid using cm_co2_tax_2020 in new diffLin2Lin and diffExp2Lin implementations. Why? 2020 is a historical year by now, and it is not intuitive to adjust carbon pricing from 2025 or 2030 via switch for year 2020.